plugin/editorconfig.vim

```diff
@@ -260,7 +268,10 @@ function! s:EditorConfigEnable(should_enable)
     augroup editorconfig
         autocmd!
         if a:should_enable
-            autocmd BufNewFile,BufReadPost,BufFilePost * call s:UseConfigFiles()
+            autocmd BufNewFile,BufReadPost,BufFilePost * call s:UseConfigFiles(0)
+            if g:EditorConfig_enable_for_new_buf
+                autocmd VimEnter,BufNew * call s:UseConfigFiles(1)
```

Comment:
I think this should be unconditional, i.e., remove lines 272 and 274.  That way, if the user changes `g:EditorConfig_enable_for_new_buf` from false to true while running, `UseConfigFiles()` will be called for new buffers.

Reply:
You're right; the function checks for it anyway